pkg/endpoint/endpoint.go

```diff
@@ -292,6 +292,10 @@ type Endpoint struct {
 	// Immutable after Endpoint creation.
 	K8sNamespace string
 
+	// K8sUID is the Kubernetes UID of the pod. Passed directly from the CNI.
+	// Immutable after Endpoint creation.
+	K8sUID string
```

Comment:
Given this is immutable after endpoint creation, should this be unexported?

Reply:
Don't we need to exported for marshaling purposes?

Reply:
Yeah, good point. I wasn't considering this struct is marshalled.